ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/PlatformCve/PlatformCvePage.tsx

```diff
@@ -36,12 +59,21 @@ function PlatformCvePage() {
         }, 1500);
     }, [cveId]);
 
+    const tableState = getTableUIState({
+        isLoading: affectedClustersRequest.loading,
+        error: affectedClustersRequest.error,
+        data: clusterData,
+        searchFilter: querySearchFilter,
+    });
+
     return (
         <>
             <PageTitle title={`Platform CVEs - PlatformCVE ${cveName}`} />
             <PageSection variant="light" className="pf-v5-u-py-md">
                 <Breadcrumb>
-                    <BreadcrumbItemLink to={workloadCveOverviewCvePath}>CVEs</BreadcrumbItemLink>
+                    <BreadcrumbItemLink to={workloadCveOverviewCvePath}>
```

Comment:
Similar question whether `workloadCveOverviewCvePath` is correct for Platform CVEs?